cmake/modules/SwiftComponents.cmake

```diff
@@ -85,7 +85,12 @@ macro(swift_configure_components)
 
     string(TOUPPER "${component}" var_name_piece)
     string(REPLACE "-" "_" var_name_piece "${var_name_piece}")
-    set(SWIFT_INSTALL_${var_name_piece} TRUE)
+    if("${SWIFT_INSTALL_EXCLUDE_${var_name_piece}}" STREQUAL "")
```

Comment:
This first part should be unneccessary; CMake already treats unset or empty variables as false.